{
  "uid" : "7693025ed9ff3fa8",
  "name" : "returns implementation",
  "fullName" : "Contract: UpgradeableBeacon once deployed.returns implementation",
  "historyId" : "a2f7c483fd6ea71a563d0c34949071b3",
  "time" : {
    "start" : 1635554972363,
    "stop" : 1635554973345,
    "duration" : 982
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067e070>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067e070>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/proxy/beacon/UpgradeableBeacon.test.js:20:42)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: UpgradeableBeacon once deployed"
  }, {
    "name" : "testClass",
    "value" : "Contract: UpgradeableBeacon once deployed"
  }, {
    "name" : "testMethod",
    "value" : "returns implementation"
  }, {
    "name" : "package",
    "value" : "Contract: UpgradeableBeacon once deployed"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "dfd5302d729e3267",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04336d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554903222,
        "stop" : 1635554904217,
        "duration" : 995
      }
    }, {
      "uid" : "c698efe6fd2f0004",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0686490>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554901901,
        "stop" : 1635554902489,
        "duration" : 588
      }
    }, {
      "uid" : "5b4547005a821bb2",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a066c730>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554890999,
        "stop" : 1635554891623,
        "duration" : 624
      }
    }, {
      "uid" : "e47069336f2aa76f",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07f4fd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554886453,
        "stop" : 1635554887043,
        "duration" : 590
      }
    }, {
      "uid" : "d20977c93bbf0eec",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a17aaaf0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554837150,
        "stop" : 1635554838133,
        "duration" : 983
      }
    }, {
      "uid" : "94374e58bcb20cbc",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a042dc10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554821459,
        "stop" : 1635554822053,
        "duration" : 594
      }
    }, {
      "uid" : "7e3f9ed66c42fe64",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0fe6730>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554808698,
        "stop" : 1635554809295,
        "duration" : 597
      }
    }, {
      "uid" : "34f2ac5f41c41baa",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be80610>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554806208,
        "stop" : 1635554806797,
        "duration" : 589
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "f9a19482f8f910b7",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/f9a19482f8f910b7",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551253277,
          "stop" : 1635551253346,
          "duration" : 69
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "7693025ed9ff3fa8.json",
  "parameterValues" : [ ]
}